packages/gasket-plugin-start/lib/index.js

```diff
@@ -32,8 +32,7 @@ module.exports = {
           name: 'preboot',
           method: 'exec',
           description: 'Any setup before the app starts',
-          link: 'README.md#start',
-          command: 'start'
```

Comment:
Shouldn't this stay?

Reply:
1. `preboot` happens during _both_ `local` and `start`. 
2. `start` is also a lifecycle, which ends up being erroneously indicated as a circular reference.